package.json

```diff
@@ -371,6 +371,12 @@
                     "scope": "window",
                     "description": "Avatar image cache expiration (0 = cache disabled)"
                 },
+                "gitHistory.sideBySide": {
+                    "type": "boolean",
+                    "default": false,
+                    "scope": "window",
```

Comment:
Should this be `application`.
I'd assume that users won't want this for all instances of VS Code, across all workspaces.
Ie unlikely someone would have sidebyside in one workspace & the default in another workspace.